cloud/shared/variable_definitions.json

```diff
@@ -260,5 +260,17 @@
     "secret": false,
     "tfvar": true,
     "type": "bool"
+  },
+  "ECS_TASK_CPU": {
+    "required": false,
+    "secret": false,
+    "tfvar": true,
+    "type": "integer"
+  },
+  "ECS_TASK_MEMORY": {
+    "required": false,
+    "secret": false,
+    "tfvar": true,
+    "type": "integer"
```

Comment:
**suggestion:** ah woops, sorry. I linked to the wrong file. This should be in the AWS-specific variable definitions: https://github.com/civiform/cloud-deploy-infra/blob/main/cloud/aws/templates/aws_oidc/variable_definitions.json

Reply:
Ahh makes sense, thank you!!